ruby_event_store/lib/ruby_event_store/mappers/encryption_mapper.rb

```diff
@@ -6,7 +6,7 @@ class EncryptionMapper
       def initialize(key_repository, serializer: YAML, forgotten_data: ForgottenData.new)
         @pipeline = Pipeline.new(
           transformations: [
-            Encryption.new(key_repository, forgotten_data: forgotten_data),
+            Encryption.new(key_repository, serializer: serializer, forgotten_data: forgotten_data),
```

Comment:
> Encryption transformation should use the same serializer as whole pipeline

Not really but we've discussed that already :)

Reply:
On a second thought maybe that's good call to use the same. If you go with JSON and schema, you'll be covered for any loss in the transformation.